### DISCUSSION TITLE
Cannot Boot Jetson Nano with the MUSHR Image

### DISCUSSION THREAD

**Discussion by brightsnowzhao** [prl-mushr/mushr#57] [Software]
Hi, I was not able to boot the Jetson with the mushr image.  It displays the default nvidia display for about 12 seconds, then goes dark for about 3 seconds, then goes back to the default display and repeat this circle forever.

It boots successfully using the Jetson image downloaded from their website.  So it looks like an issue with the mushi image.  I tried reburning it three times and continued seeing the same problem.  I also tried the Jeson image a second time and it worked again.

I was testing the Jetson nano out of box.  Only connected it to the micro SD card, a monitor, a USB mouse, and using the 5V DC power (not the battery).  I haven't connected it to any other devices.  

Can you please advise if I missed something?  E.g. do I have to connect the network card or other hardware before booting it?

**Comment by palanc** [comment]
Hi @brightsnowzhao - Sorry about that, I think that the system image is no longer compatible with the latest hardware version of the Jetson Nano. However, I just uploaded an updated image to the website that should now work with the latest hardware version. Could you please try downloading the image again using the instructions given here: [https://mushr.io/hardware/build_instructions/#sd-card-setup](https://mushr.io/hardware/build_instructions/#sd-card-setup)

Alternatively, you could start from the stock image and then installing everything using a script by following these instructions: [https://mushr.io/hardware/nano_install/](https://mushr.io/hardware/nano_install/)

Please let us know if you try this and are still having issues.

**Reply by brightsnowzhao** [reply]
Hi @planc509 - thank you!  I'll try the new image later.  Haven't installed the network card yet, so the image is easier.  I'll let you know the result.

**Reply by brightsnowzhao** [reply]
Hi @planc509 - the new image works.  I got to the log-in page.  Thank you!

BTW, I notice that the new image is much smaller (15GB vs. over 30GB, decompressed).  Does it have all the same functionality?

**Reply by palanc** [reply]
Awesome glad it's working! Yes it has the same functionality, I think the old image just didn't remove the allocated null space that is copied when creating an image.
